nautobot_data_validation_engine/__init__.py

```diff
@@ -18,7 +18,7 @@ class NautobotDataValidationEngineConfig(PluginConfig):
     version = __version__
     author = "Network to Code"
     author_email = "opensource@networktocode.com"
-    description = "Plugin that provides a UI for managing custom data validation rules."
+    description = "Nautobot App that provides a data validation rules engine to easily codify data standards in the UI. Includes RegEx and Min/Max validation rules."
```

Comment:
I'd rather not list specific rule types supported by this app here, as it'll be inaccurate once #29 gets merged unless we update it again then.

Reply:
Fixed